docs/samples/tasks.yaml

```diff
@@ -65,20 +65,28 @@
   # authenticate leader-to-helper requests. In the case of a leader-role task,
   # the leader will include the first token in a header when making requests to
   # the helper. In the case of a helper-role task, the helper will accept
-  # requests with any of the listed authentication tokens. Each token is encoded
-  # in base64url, and the decoded value is sent as an HTTP header value. For
-  # example, this value decodes to
+  # requests with any of the listed authentication tokens.
+  #
+  # Each token's `type` governs how it is inserted into HTTP requests if used by
+  # the leader to authenticate a request to the helper.
+  #
+  # Each token is encoded in base64url, and the decoded value is sent in an HTTP
+  # header. For example, the first token's value decodes to
   # "aggregator-235242f99406c4fd28b820c32eab0f68".
   aggregator_auth_tokens:
-  - "YWdncmVnYXRvci0yMzUyNDJmOTk0MDZjNGZkMjhiODIwYzMyZWFiMGY2OA"
+  - type: "DapAuth"
+    token: "YWdncmVnYXRvci0yMzUyNDJmOTk0MDZjNGZkMjhiODIwYzMyZWFiMGY2OA"
+  - type: "Bearer"
+    token: "YWdncmVnYXRvci04NDc1NjkwZjJmYzQzMDBmYjE0NmJiMjk1NDIzNDk1NA"
 
   # Authentication tokens shared between the leader and the collector, and used
```

Comment:
Just curious: were there any thoughts about authenticating leader->collector and leader->helper communication using mutual TLS? Just thinking it would simplify authentication logic, insofar as we don't have to compare tokens ourselves. Not saying we should do that, moreso wondering if the conversation ever came up.